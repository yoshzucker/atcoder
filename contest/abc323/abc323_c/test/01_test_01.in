100 100
2500 1400 1700 2500 1400 2200 800 2000 1400 1100 500 700 2500 1500 1000 500 2000 2200 700 700 1200 500 2100 1600 900 2200 2000 1100 1500 1500 1800 700 500 800 1900 1000 1900 1300 2300 1600 2000 2000 1900 1200 1000 500 1700 2400 1900 700 2400 1400 1300 800 2300 800 900 1600 1600 1500 700 2200 800 1700 2400 1800 600 2300 1400 1900 2000 500 2100 600 800 1000 700 2200 2100 800 500 1800 500 1800 1900 1600 1300 2400 2100 1000 700 900 1900 1800 1100 700 2500 1600 1800 1500
xooxoxxoooxxxoxxoxoooxooxxoxooxxxxoxooxxoooxxoooxoxooooooxoxoxxxoxooooxxoxxooxxxxoxxxoxxoxxoooxoxxox
xoooxxxxoxxooxoxoxooxxooxxxxxxoxooooxooxooxxoxxxoxxooooxxoxooxxoxoxooxxoxxxooxxoxoxooxxoxxxxoxoxoxox
oxooxxxxoxoooxooxoxoooxoxxxxoxoxooxxxooxxxoxoxooxxxxoxxxoxoooxxoxxooxoxooxooxooxxxoxxxooooooooxxxoxo
xooooxooxoxxoxoxooooxooxooxxxxoooxxooxxxooxxoxxoxxoooxxoxxxoooooxxxxxooxxxxxoxooooooxoooooxooxxoooxx
oooxxxxxooooxxoooooxxoxoxoxxxxxxxooooxxxooxooooxxxoooxxxooxxoxxxoxxoxxxxxoxoxxoxoxoooxxxxxoxxooxoxox
xxxxoxxxxooxxxxxoxoxooooooxxooxooxxoxxooxoxoxoxxoxxxxxoooooxoooooxoooooxxxxooooooxoxooxxxxxxoooxxxoo
oxxoxoxxoxxxxxxoxoxoxxxoxxxxooxxoxoxoxooooxxoxxxxooxxoxoxooxoooxooxoooooxoooxxoooxoooxoxxxooxxooooxx
oxxxxxoxooxxxxoxoxxoooxoooxoooxxxoxxxooooxxxxoxoooxoooxxoooooxxoxxxxxxxxoooxooxxoxxxxxooxxxxoxxooxoo
xoxooxoxooxxooxoooxoxooxooxoxoooxoxxoxoxxxxoxxxxooxxoxoxxxxxoxoxxoxoxoxooxxxoxooxxxxxooooxxxxoxoxoxo
oxxxoxxxxxxxxxxxxoxoxoxooxxxoooxxxooxxxooxooxoxoooxxxooxxooooxxoooxxoxxoxxxxoxooxxooooxxoxoxxooxoooo
oxooooxxoxoxxooxoooooooooxxoooxxxxoxxxxoxxoooooooxxxxoxxooxxxoooxxoxxoxooxxxooxxxoxooxxxxxxooxxxxooo
oxooxxxxxoxoxxxxooxxoooxoooxxooooxoxxooxoxxxooxxxxoxoxoxoooxxooooxxxxooxoxxxxxxxxooxxxxooooxxooxxxox
oooxxoxoxoooxxxoxoxxoxooxoooxxoxxoxooxoxoxxxooooooxoxoooxxxxooxxxxoxoxxooooxoxxoxooxoxxxooxoxxxxxoxo
xoxxooxxxxoxxxooxoxoooooxoxxxooxxxxoxoxxooxxoxxxooxooxxxoxoxxooxoooxxoxoxxooxooooxoxxxxoxxoooxxxxxxo
ooxooxxxoxoooooxxooooxxoxxooxxooxxxxxoxxxxxxooxxoxxxxxoxoxoxoxoxxoooooooxoxooxoooxxxxxooxoxxxoxooxoo
xxxoxoxxxxoooooxxoxoxoxxooxoxooxxooooooooxxxxxoxoxxxoxoxxxxxooxxooxxoxooxoxoooxxooxoxoxxoxoxxoxxxxox
xxooooxoxxoxxxoxxxxoxxooxoooxoxoxxoxoxxxoooxxxooooxxoooxxoxooooxxxoxoxxxoxxxxxxxoxxooooxxxxooxoooxxo
oxxxxooooxooooxxxxoxxxoooxxxooooooxoooxooxoxxxoxoooxoxooxooooxxoxxxoxxooxoxxxxoxxooxxooxxxxxoooxooox
xoxxoxxooooxoxooooxooxoooxxxxxxoxxoxooxoxxxxxoxoxxxoooxooxoxoxooooxoxxooxxxooxxooooooxoooxxoooxxxoxx
oxoxoooxxoooxoxxxxxooxxooxxxxoooxxxxooooxxxooxxoooxoxxoxoxoxoxxooxoxoooxooxooxoxxooxxxoxooxoxxoooxox
xoxxxxooxxoxoxxxoxoxxoxoxoxxooxxxooxoxxoxoooooxxxxxoooooooooxxxxooxxxooxxoxoxoooxoxoxxooooxoxoxxxoox
ooxxoxxooxoooooooxoooxoooxoxooooxoxxoooxooxxxxxxooxxoxoooxooxxxxoxxoxoxxxoooooxxoxxooxxooxxooxoooxoo
oxxxxxooxooooooxoxxxxxoxxxoxxxoxoxoxxxxxoxoooxxooxxxxxooooxxoxxxxxoxxooxxxooxooooxoxooxoooxxxxxxxxoo
oxoxoxxxoxxoxxooxxoxoxxxooxoxooxxxxoooxxxoooxxooxoxooxxxxxoooxxoxoxoxoxxooxoxoxxoxoxoxxooooooxoxxxox
xxxxoooxooxoxxooxoxxxxoxxxxooxxoxoxxoooooxoxoxoxoxxoxooxoooxxoxxoooxoooxxoooxxooxoxxoxoooxooooxooxxx
oxooxooxxxoxoxoxxoxxxoooxxxxooxxxxooxxxoxoxxoxoooooooxxxxoxoxxxooxxoooxxoxooxooxxxoxxxoxoooxxoooxoxx
ooxxxoxooxxxoxxooxoxxxoxooxoooxooxxoxxoooxxoxxxxoxooxooxoooooxooxxxxxxoxxooooooooxoooooxooooooooxoxx
xxoxxoxoooxooooxoxoxxoxooxxooxxooxoxxxoxxoxxoxxooxooxxxooxoxxxxooxoxooxxooooxoooooooxoxoxxoxooooxoxx
ooooxooxoxxxxxooxxoxooxoxxoooxoxxoxxoxooxxoooxxxxooxxxxxooooooxxoooxxoxxoooxooxxoxxxoxxxxxooxxoxoooo
ooxooxoooxxoxooxoxxooooooxooxxxoxxxxoxooxxxxxxoxoxxooxooxxooxxooxxxxxoooooxxoxxooxxoxooxoxoxxxoxooxo
ooxoxoxooooxooxxoooxoxooxoxxxxxxooooooxoxoxxxoxxoxxxooxoxoxooxxxoxxoooxoooooooooxxxooxoxxxxxxoxooxoo
xooxoxooxxxxxxooooxoxxxxooooxooooooxxxoxoxoxooxxoxoxxxxoxooxxxxxooooooooooxoxoxoooooooxooxxooxoxxoxx
xxoooxxoxooooxoxxoxooxooxoooxxxooxxxoxooxxxxxoxoooxxxxoxxoxxxxoxxoooooxxxxxxxxoxxxoxoxxoooxxxxoooooo
xoxxxooooxoxxxooxxoooxxoxxoxxooxoooooooxxxooxooxoxxxoxooxxxoxxxxxooxooxxxxooxxxxxxoxxoooxxxooxooooox
oxoxoxooxoxooxxxxxxxxxxooxxxooxooxoxxoxxoxooxooxoooxoxoooxxxooxooxxxoxoxooxoxoxxxooxxoooxxoooooooxoo
xoxxoxoxoxxxxoxxooxxxooxoxxxoxxoxxxxxoxxxooxooxxxoxxxxoxxxoxoxooxxoxxooooooxooxoxoxxxoxooxxxxxxxxxox
oxxxxxoooooxooooxxxoxooxxxoxxxoxxooxoxxoxoooxoooxxoxooxxxoxoxxxxxoooxxxoxoxxxooxxxxooxooooxxxoxooxxo
ooxxxoooooxoxxxooooxoxooxxoooxooooxxxxoxoxxooxoxooooxoxxooxxoxoxxoxoxoxooxoxoxxooxoxoooooxxoxxxxoooo
oxxooxxxxxoxxxoxxxxxxxooxxxoxoxoxoxxoxoxoxxooxxxooxooxoxoxxxooxxoooxoxxoooxxxxxxxoxooxooxxxxxooxxoox
oxooxoxooxxxoxoxxxxxoxxooooxxoxooxxxoxxxoxooxxooxoxoooxoxxxoxoooxxooooxxoxoxxoxoxxoxooxoxxooxxoxxoox
xoxooxxooxxxooxoxoxooooxxoxoxoxoxxxxoooxxxxxooxxoxoxxxxooxxooooxoxxoxxoxxooxoxoxoxoooxxxxoxxooxoxxox
ooxxoxoxoxoooooxxoxoxoxoxooxxoooxxxooooooxoooxxoxxoxxxoxooxxooxxoxxxxxxxooxoooxoxoooooxooooxxxooooxx
ooxxooxxxxoooxoxxoxoxoxxoooooxxxoxoxoxoxooxooxoxoxxoxxxxxoxxoxxxxoooxoxxxxooxxoooxoxoxooxxxoooxoooox
xoxxooxxxooxxxxoxxxoxoxxxoxooxxooxoxooooxxooxoooxxoooxoooooxxxxxxxoxxxxooxoxxoxoxxoxoooxxoxoxxxxxxox
oooooxxooxxoxxoxxoxooxooxoxoooxxoooxoxxxooxxxxxoxooxooxxxoxxooxoxxxoxoooxoxxoxoxxooxooxxxoxxoxooxxox
xoxooxxooooxooxooooxxxxoxoxxxooooxxxoxoxooxooxoooxxoxoxoxoxooxxoooooxxxxxoooooxoxoxoxoooxxoooxxoooxo
xoooxooooooxoxoxooooxxoxxxxoxxoxxoxooxxxoooxxoooooooxxoooxxoxoxxoxxoxoxxoxoxoooooxooooxxoxoxxxooooxx
xoooooxooxxxxxooxoxoxoxoooxxxxxxooxxoxooooxxxxxoxxoxxoxxoooxoxxxoxxoxxoxoooxoxoxooxoxoxoxooooxoxxxox
xooxoxxooxoxooxxxxooxxxxxoxxoxxooooxxxooooxxxxoxoxooxxxxoxooooxxxoxooxxxxxxooxxoxoooooxoxxxxooooxxxo
xooooxoooxooooxooooooxxxoxoooxooooxoxoxxxoxoxoxxoooooxoxoxxoxoxxoxoxooxoxoxxoxooxxoooxxoxxoooxxxooox
oxoxxxxxxxxxxxooooxxxoooxxoooxoxxoxxooxxoxoooooooxooxooxooooooooooxxooxxxooooxxoxooooooxxoooxxooxxoo
oxoxxooooxoooxoxoxoxooxxooxooooxoxxxxxxoxooxxxxxxxxooxxxxxxoxxxxoxoxxoooxxxxoxxxooxoxxxxoxxooooxxoxx
xxoxxoxxoxooxoxxxxxxoooxxxoxooxooooooxoooxxoxoxoxxxxxxooxoxxoxxoxxoxooxoxoxxxxxooxoxoxxxoxooooxxoooo
xxxxoooooxxxooxxooooxxxoxoxoxxxoxooxooooxxxxoxoxxoooxooooxooxxxoxooxoxoxxoxxxoxoxoxooxxxxxxxooxooxoo
oxxoxoooxxxoxxxooooxxxooxxoooxxoxoxxxxoooxxooxxoxxxoooxxxxoxoxxoxxooxxoxoxooooxxoxxoooxoooxxxoxxoxxo
oxooooxoxxxooxoooxxxooxxxoxxxoxooooxxxxoooxxooxxxoxoooxoooxoxooxooooxxxooxxoooooxoxxxoooooxooooxxoox
xxxoxoxooxxxxxoxxoxxooxooxooxxooxxoxxxooxxxoxxxoooooxxxxoxxooxxxxooxxoxxxxxoxxooxoxoxoooxoxoxoxxooxx
ooooxooxooxoxoxxxxoxxxoxooxxxxoxooxoxoooxooxoxoxooxooxxxxooxxooxxoxoooxooxxxoxxxoxooxooooxoxooxxoooo
xoxxxooxoxxoxooxxooxxxxooxxxoxxoxoxxooxxooooxxxooooxoxxoxxxxxxxxxooxxxxxoooxxxoxooooooxooxooxxooooox
oxxxoxxoooxooxoxoooxooooxooxoooxoxxxoooooooxxooooxoxxooxoooxoxxxxoxxoxoxooxoxxoxoxoooxoooxxoxxoxxooo
xooxxxoxxooxxxxxxoooxoxoxxoxxxxooxoxxxxxxxoxxoxxooxxoxoooxoooooxxooooxxoxoxxxoooxxoooxxxxxoooooooxxo
oxoxoxoxooooooxoxxoxooxooxxooxxxoxoxxoxxoxooxxoxxoxoxxxxoxxxooxoxooooooxxxxoxoxxxxoxoxxxoxxooxooxxxo
xooxooxxoooooxxxxoxoxooxoxoxxxxxxoooxoxoooxxoxooooxxooooxoxxooxoxoooooooooxxxxoooxxxxxxxxoxxooxoxxox
xxooxooooxooxoxoooooxxxoxxxxxoxoooxxxxoxoooxooxoxooxooooxoooxxooxxooxooxxooxoxoxoxxxooxxxoxoxoxxxoxx
xoxoxoxoxoooxxxoxooooxxxxooxooooxoxoxxxxxxxoxxxoooxooxoxxooxxxxoxxoxoxxxooooooooooxxoxoxooooxoooxoxo
ooooxxoooooxxxoxxoxxxooooxxoxxxoxxooxoxooxxxxxxxxooxoxooooxooxoxoxxooxoxxooxxxxooooxoxxooxoxooooooox
xxoxxooxooooooxxxxxoxoxxxoooxooxxoxxoxxxoooxooxxxoxxxxoxooxxxoxxooxoxxxxxxxoxoxoxooooxxooxxxxoxxoooo
xxxxoxooxooooxoxxoooxxxoooxxoxxxoooxxxoxxoooooxooxxxooxxooxooxxoxoooxooxoxooxooxxooooxxxxoxxoxoxoxxo
xxoxoxoxoxoooxooxxooooxoooxoxooxooooxooooxxxxoxxxoxxoxxxoooxooooxxxoxxooxxxxoxoxooooxxxxoxxoxxxoxxxx
xxooxoxooooxxoxxxxoxxxoxoxxxxoxooooxxoooxoxxxxooxoxxxxxxoxxoxxooooxxoxoooxxooxxxoooxxoooxooxxooxooxx
ooxxoooxxxoooxxooooxxoooooxxoxooxooxxxxxoxooxoxxooxxxxxoxooxoxxooxooxoxxxxxxooooxooooooooxxxoxxxoxxx
xxooxoxooxxxxoxooooxxooooxoxxxxxooxxxxxxxxxxxoooxxxxxoxxoxxxoxxooxxoooooxxoxooxooxxooxooxxooxooxxxox
oxoxxxoxxooxoxxoxoxxxxxxoxooxooxxxxooxoxoxxoxoooxoxxxxoooxxxxxxxxxoxoxxxxxxooooxoxoxxoxxoxoxoooxxoox
xxxxxoooxooxoxxxxooxoooxxxoxoxxooxoxoxoxoooxoxxxxxoooxxxooxooxoxooxxxxoooooxxxoxxoxxxoxxxxoooxxoxxoo
ooxxoxxxxxoxxoxooxxxxooooxoxxoxxxxxoxooooxoxoxxxxxxxooxxoooxxxoooxxoxxxxxxoxoooooxoxxxxxoooxoxxoxoox
oooxxxoooxoxxxxoxoxooxoooooxoooooooooooxoooooxoooxxxooooxxoxoxxoooooooooooooxoxxooxxxoxxoxoxxoxxxoox
oooxoooxoxxooxxoxxoxooooooxxxooooxxxoxxoooxoooxooxoxxooooxoooxxoxxxxoooooxoxxxxoxxooxoxxxooxxooxooxx
xooxoxxxoxooxoxoxoxoxxoxxxxoxxoxxoxxxoxxoooxoooooxxoxooxoooxooooxoxoxxxooxooxxxxoooxxxoxxoooxxxxxxox
oxxoxoooooxxxooxxooxooooooooxxxxxoxooxooxoooooxxxxooxoooxxxxxoxoxoooxooxoxoooxooxxooxooxoooxxxoxooxo
oxooxoooxoooxoxoxxxxooxxxxoxoxoxoooxoxxxxxxxooxxoooooxoxoooooxxoxoxoooxxxoxoooxxxxxoooxxoxxoxoxooxxo
xooxooxooxxoxoooooxoooxoxxxoooxoxoxooxxxxxxxxooxxxoxoxxooxxxxoxoxoxoxoxxxxxoxooxoxxooxxoxoxoxxxoooox
ooxxooxxoooxxoxxoxxoxooxxxxxoxxxoxxoooxooooxoxoxooxoooxxxoxxoooxxxxooxoxxxxxoxoxooxooxoooxxxxoxoxxoo
xxooooooxxoxxxxxoxxoxxooxoxoxoxoxxxooxoxoxoxxxoxxoxoxooooxxoooooxooxxooxoxooxxoxxxxxxxoxooxooxxxoooo
ooxoxooxxoxxxoxxoxxooxxoxoxoooxoxoooxoooxoooxoxoxoxoxxxooxoxxooxooxxxooxxxoxxooxoooxxoxoooxooooxoooo
oxxxxoxxoooxxxoxxoxoxxxoxoxxoxoooxxxxooxxoooxooxxooxxoooooxooooxxoxxoxxoooxxooxxxxoxxxooxooooxooooox
ooooxoxoxxxooxxxoxoooxxxoooxoooooxxoxooxxoxxxooxooxxxxxxooxoxxxooooxxoxxoooxxxoxxxxoxooxooooxooooxxx
xxxxoxoxoxxxoxoxoxxoxoxoxxxxooxoxooooxxoooxxooxoxxoxxoxxxooxoxooxxoxooxxxoxxxoooxoxooxoxxoxooxooxoox
oxxoooooxoxooxoxxoooxxoxxooxxoxxoxoxoxooxxoooxxxxxooxoxxxoxoooxxoooxxxxooxxoxoooooxoxoxoxoxxoooooxox
oxoooooooooxxoxxoxooxxxoxoxoxxxxxxooxxoooxxxoooxxxxoxxxooxxxooxxoxxoooooooooxoxxooxxxxxxooooxoooxooo
oxooooxoxoxxxxoxxxooxxoooxxooxxoxooxoxooxxoxxxooxxxxxooxooooxooxxxxoxoxoxoooxooxooxoxoxxxoxxxxoxxoxx
xxooxxoxxxoxoxoxoxxxoxoxoxooooxxxoxoxoxxxxooooooooooooxooxoxooooxxxoxoxooooooxxxxoxoxooxoooxxoooxooo
xooooooooooxoxxoxxoxxxooooxoxxoxooxxoooxoxoooxooxxxoxxoxoxoxxxoxooxoxoxxooxxxxooxoooxxoxxxoxxoxxxxoo
xoxoooxxxooxxxxoxxxoxxxoooxooooxooxxxxxoxoooxxooxoxoxoxxooxooxooxooxxxoooxxoxoxxxooxooxxoxoxxoxxxooo
xxoxoooxxxooxxxooxxoxxooxoxooxxoxoooooxxoxxoxoxoxoxoxxxxxooooxooxooooxooooxooooxoxxoooxxooxooxxxxoox
oxxxoooxoxoooxxxoxxxooxxxoxxxoooxxoxxxxoxxooxxxxoxoooxoxxoxoxoooooxoxoxxoxxxxoooxxoxoooooxoxoxoxxooo
ooooooxxxxxxoxoxoxooxoxxooxxoxooxoooxoxooxoxxooxooxooxxoooxxoooxxoxooxxxoxxooooxooxxoooooxoxoxoxoooo
ooxxxxxooooxoooxoxxooxxxoxoxoxooxooooxooxoooxoxxxoxxxoxxoxxxoxxxxxxxooxxooooxoxxoxoxoxoxxooxoxoxoooo
xoxxxxxooxoxoxoooxxxoxxooxoxxxxxxoooxoxxxooooooxxooooxoxoxxoxoxoxxoxxoxxxooxoxxxxoxooxoooxxoooxxoxox
oxxoooxxoxxoooooxxoxoxoxxoooooxoxxxxoxxoxoooxxxooxooxxxoooxoxxxxxoxoooxoooooxxooxooxxoxoxoxxoxoooxxx
xxxxxooxoxooxxxoxooxxoxoooxxxxxoxoxoooxxxxxoooooooxxxoxxxooxxooxxoooooooxxxooxxxooxxooooxoxxoxooxoxx
