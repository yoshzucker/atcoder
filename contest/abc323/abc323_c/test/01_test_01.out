8
11
10
6
14
10
10
14
12
11
9
11
11
15
12
11
10
7
9
10
9
4
14
12
7
9
6
8
10
10
6
6
11
12
8
19
10
6
16
10
13
5
10
16
9
4
4
10
11
6
5
15
13
9
10
5
13
5
12
2
15
13
8
10
9
8
11
8
14
12
8
14
17
12
15
0
7
13
3
9
11
10
9
4
11
8
12
5
8
11
7
7
11
8
7
3
11
11
7
13
